### DISCUSSION TITLE
[remark-stringify] Is there a way to that remark-stringify use original file options

### DISCUSSION THREAD

**Discussion by rluvaton** [remarkjs/remark#769] [Q&A] [answered]
For example, if I write bullet points in markdown sometimes using `*` and sometimes using `-` is there a way for `remark-stringify` to keep that format?

I'm trying to change some content in one node after using `remark-parse` and when using `remark-stringify` it changes the whole markdown

**Comment by wooorm** [comment, marked as answer]
No, there is not. You can configure those options yourself though.

**Reply by ChristianMurphy** [reply]
To add on to this.
Remark/MDAST provide an abstract syntax tree. This keeps the structure of the document (lists, links, headers, etc), but not the exact syntax used `-` vs `*` vs `+`.
In many cases the exact syntax is not needed (a list item is still a list item no matter which marker is used), and is easier to work with.

But if you specifically need the exact syntax, a lower level of abstraction https://github.com/micromark/micromark provides the specific tokens and could be used to create a concrete syntax tree (which keeps exact syntax information).

**Reply by rluvaton** [reply]
Thank you very much!

I see `micromark` only output HTML, is there a way to integrate it with this library? Or I need to create an extension for `micromark`?

Thank you so much for your help
